internal/buildscripts/packaging/installer/install.sh

```diff
@@ -28,6 +28,13 @@ get_distro() {
   echo "$distro"
 }
 
+get_distro_like() {
```

Comment:
To avoid redundancy with the installation/uninstallation steps below, maybe we can just update the `get_distro` function for `ID_LIKE`?  Something like:
```
 get_distro() {
   local distro="$(. /etc/os-release 2>/dev/null && echo ${ID:-} || true)"
+  local distro_like="$(. /etc/os-release 2>/dev/null && echo ${ID_LIKE:-} || true)"
+  case "$distro_like" in
+    *rhel*)
+      distro="rhel"
+      ;;
+  esac
 
   echo "$distro"
 }
```

Reply:
Good idea! I agree with your proposed changes. There is less code that has to be changed and less duplicate code